android/translate/app/src/main/java/com/google/mlkit/samples/translate/java/TranslateFragment.java

```diff
@@ -76,8 +76,7 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         final Spinner sourceLangSelector = view.findViewById(R.id.sourceLangSelector);
         final Spinner targetLangSelector = view.findViewById(R.id.targetLangSelector);
 
-        final TranslateViewModel viewModel =
-                ViewModelProviders.of(this).get(TranslateViewModel.class);
+        final TranslateViewModel viewModel = ViewModelProviders.of(this).get(TranslateViewModel.class);
```

Comment:
Unrelated to this PR, but `ViewModelProviders` was deprecated in favor of [ViewModelProvider](https://developer.android.com/reference/androidx/lifecycle/ViewModelProvider).

Reply:
There are **a lot** of other things to improve.
I see a better chance to get merged by doing it step by step